client/.husky/sort-json.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Script to sort specific JSON files
+
+TRANSLATION_FILES=($(find ./client/packages/common/src/intl/locales/* -type f \( -name "*.json"  \)))
+
+for FILE in "${TRANSLATION_FILES[@]}"; do
+  if [ -f "$FILE" ]; then
+    cat "$FILE" | jq --sort-keys >> output.tmp && mv output.tmp "$FILE"
```

Comment:
Does this require `jq` installed?
I wonder if we should base it on the one from HSH instead?
https://github.com/msupply-foundation/health-supply-hub/tree/develop/frontend/.husky

Reply:
Is this not being moved to actions?

Reply:
@fergie-nz  can you create an issue for creating an action for this?

Reply:
https://github.com/msupply-foundation/open-msupply/issues/4971